# offset board_t: [-26.0, 0.0, 13.0]

board_t: [106.0, -3.0, 0.0]

vessel_ends: [     60.7, 20.2, -3.0,
                    60.7, 60.2, -3.0]

object_h: 5.0


object_d: 2.0


